apis_ontology/api/serializers.py

```diff
@@ -40,7 +40,7 @@ class Meta:
 
 class WorkPreviewSerializer(serializers.ModelSerializer):
     expression_data = ExpressionDataSerializer(required=False, many=True)
-    work_type = WorkTypeDataSerializer(required=False, allow_null=True)
+    work_type = WorkTypeDataSerializer(required=False, allow_null=True, many=True)
```

Comment:
I think the `allow_null` argument needs to be removed with the switch to a list.

It could theoretically be replaced with `allow_empty`, but that's redundant when set to `True`.